apis/python/src/tiledbsoma/tiledb_object.py

```diff
@@ -125,6 +136,80 @@ def exists(self) -> bool:
         except tiledb.cc.TileDBError:
             return False
 
+    _open_mode: Optional[str] = None
+    _open_timestamp: Optional[int] = None
+
+    def open(
```

Comment:
Timestamps should be a tuple/list for `[start, end]`.

Reply:
@thetorpedodog @bkmartinjr @johnkerl @Shelnutt2 Are we ok with using timestamp *ranges* at the SOMA API level? Thought I'd double-check as the concept didn't come up in https://github.com/single-cell-data/SOMA/issues/44#issuecomment-1332780970 and strikes me as quite specific to advanced TileDB capabilities. 
Should we say the timestamp type is `Any` ("token") at the somacore level and `Tuple[int,int]` in the TileDB implementation? A simpler approach might be to take `timestamp: int` and take that to mean `(0, t)` when we call into TileDB. But perhaps `int` isn't general enough at the somacore level, like the example of using a commit digest?

Reply:
Another idea from @johnkerl, take *two* arguments with defaults `start=0` and `end=[now]`.